app/views/categories/new.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="new-category">
+  <h2>New Category</h2>
+  <%= form_with model: @category, url: categories_path do |f| %>
+    <%= f.text_field :name, placeholder: 'Enter Full Name' %>
+    <br>
+    <br>
+    <div class="field">
+      <%= f.file_field :icon, placeholder: "Icon", class: "form-control stuff" , required: true%>
+    </div>
+    <br>
+    <%= f.submit('Add Category') %>
+  <%end%>
+</div>
```

Comment:
- [ ] Instead of doing nothing when a user submits invalid details, kindly present the user with the error messages.

- [ ] This Category form should have a Back (<) button for navigating back to the Category page. This is important in case a user decides not to create a category anymore.

Reply:
Fixed